README.md

```diff
@@ -1,6 +1,12 @@
-This is a Formula Student Driverless Simulation Competition System.
+This is a Formula Student Driverless Simulation Competition System. (FSDS)
```

Comment:
hey! it stuck!

Reply:
A bit of a tongue twister, don't you think? 😄

Reply:
YEA i sort of like it